### PR TITLE
Add Edge iOS and Edge Android browser detection

### PR DESCRIPTION
**Pull request by joaocac**
Add Edge iOS and Edge Android browser name detection based on the information from Microsoft.

https://blogs.windows.com/msedgedev/2017/10/05/microsoft-edge-ios-android-developer/

### CODE REVIEW

**Comment by jdalton** [comment]
Hi @joaocac!

Would you add a tests too?

**Comment by joaocac** [comment]
Yes, I will.
I will update the PR soon

File: platform.js [hunk 1]
```diff
@@ -695,7 +695,7 @@
     // Detect non-Opera (Presto-based) versions (order is important).
     if (!version) {
       version = getVersion([
-        '(?:Cloud9|CriOS|CrMo|Edge|FxiOS|IEMobile|Iron|Opera ?Mini|OPiOS|OPR|Raven|SamsungBrowser|Silk(?!/[\\d.]+$))',
+        '(?:Cloud9|CriOS|CrMo|Edge|EdgA|EdgiOS|FxiOS|IEMobile|Iron|Opera ?Mini|OPiOS|OPR|Raven|SamsungBrowser|Silk(?!/[\\d.]+$))',
         'Version',
```
Comment (jdalton) [inline platform.js]: Is `EdgA` the full identifier?

Comment (joaocac) [reply platform.js]: From Microsoft Info the UA for Edge For Android is like: ... EdgA/41.1.35.1
And for iOS is: ... EdgiOS/41.1.35.1

Comment (jdalton) [reply platform.js]: Thank you!

**Pull request merged**